nextflow.config

```diff
@@ -38,6 +38,28 @@ profiles {
     process.container = 'quay.io/nextflow/rnaseq-nf:v1.1'
     docker.enabled = true
   }
+
+  singularity {
+    process.container = 'quay.io/nextflow/rnaseq-nf:v1.1'
+    singularity.enabled = true
+    singularity.autoMounts = true
+  }
+
+  apptainer {
+    process.container = 'quay.io/nextflow/rnaseq-nf:v1.1'
+    apptainer.enabled = true
+  }
+
+  conda {
+    conda.enabled = true
+    conda.channels = 'conda-forge,bioconda,defaults'
```

Comment:
As with container, could just be in the top scope once?

Reply:
Same with this - might be better practice to set these in profiles to isolate the `conda` settings when `docker` / `singularity` .... are being used.